addons/binding/org.openhab.binding.magentatv/READMEbeta.md

```diff
@@ -0,0 +1,14 @@
+**Installing the bundle**
```

Comment:
this can be removed when the pull request was merged. This information is required at the moment and multiple users trapped already into this. The file is named READMEbeta.md to indicate that this information is only relevant for beta versions and not the regular README.

Reply:
Add a .gitignore file (that ignores itself) and add this file :) But why should something first be added and then removed again. That only increases the git index and will stay forever.